### PR TITLE
Update string_test.py

### PR DESCRIPTION
**Pull request by mhmerrill**
fix pytest skip of test_ends_with

### CODE REVIEW

**Review by hokiegeek2 — APPROVED** [review]

**Comment by ronawho** [comment]
I missed this before, but looks like `run_test_index` / `test_index` should be updated too

File: tests/string_test.py [hunk 1]
```diff
@@ -321,7 +320,8 @@ def test_contains(self):
     def test_starts_with(self):
         run_test_starts_with(self.strings, self.test_strings, self.delim)
 
-    def test_ends_with(self):
+    @pytest.mark.skip(reason="awaiting bug fix.")
+    test_ends_with(self):
```
Comment (ronawho) [inline tests/string_test.py:324]: Missing `def`

Comment (mhmerrill) [reply tests/string_test.py]: fixed

File: tests/string_test.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def run_test_unique(strings, test_strings, cat):
     assert(akset == npset)
     return akset
 
-@pytest.mark.skip(reason="awaiting bug fix.") 
+@pytest.mark.skip(reason="awaiting bug fix.")
```
Comment (ronawho) [inline tests/string_test.py:44]: Move to test_index?

Comment (mhmerrill) [reply tests/string_test.py]: fixed

**Review by ronawho — APPROVED** [review]

**Comment by hokiegeek2** [comment]
there's now a problem with test_index on MacOS:

image

**Comment by ronawho** [comment]
> there's now a problem with test_index on MacOS:

This PR already fixes the pytest.skip for test_index.

**Pull request merged**